connector-sdk/jackson-datatype-feel/src/main/java/io/camunda/connector/feel/jackson/AbstractFeelDeserializer.java

```diff
@@ -46,6 +49,9 @@ protected AbstractFeelDeserializer(FeelEngineWrapper feelEngineWrapper, boolean
     super(String.class);
     this.feelEngineWrapper = feelEngineWrapper;
     this.relaxed = relaxed;
+    BLANK_OBJECT_MAPPER
+        .registerModule(new Jdk8Module())
+        .registerModule(new JacksonModuleDocument(new DocumentFactoryImpl(null), null));
```

Comment:
Curious, as this is the deserializer. Why would it need to be document aware?

Reply:
It's due to the `WebhookHttpResponse`that we're creating in `InboundWebhookRestController#buildSuccessfulResponse`